### PR TITLE
Fix some Scrutinizer issues

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

File: src/Bridge/Doctrine/Orm/Metadata/Property/DoctrineOrmPropertyMetadataFactory.php [hunk 1]
```diff
@@ -56,7 +57,13 @@ public function create(string $resourceClass, string $property, array $options =
         foreach ($identifiers as $identifier) {
             if ($identifier === $property) {
                 $propertyMetadata = $propertyMetadata->withIdentifier(true);
-                $propertyMetadata = $propertyMetadata->withWritable($doctrineClassMetadata->isIdentifierNatural());
+                if ($doctrineClassMetadata instanceof ClassMetadataInfo) {
+                    $writable = $doctrineClassMetadata->isIdentifierNatural();
```
Comment (theofidry) [inline src/Bridge/Doctrine/Orm/Metadata/Property/DoctrineOrmPropertyMetadataFactory.php]: ternary operator instead?

**Comment by theofidry** [comment]
👍

**Comment by teohhanhui** [comment]
:+1:

File: src/Bridge/Doctrine/Orm/Metadata/Property/DoctrineOrmPropertyMetadataFactory.php (hunk 1 above)
Comment (dunglas) [inline src/Bridge/Doctrine/Orm/Metadata/Property/DoctrineOrmPropertyMetadataFactory.php]: Too long line.

File: src/Bridge/Doctrine/Orm/Util/QueryChecker.php [hunk 2]
```diff
@@ -67,12 +57,17 @@ public static function hasRootEntityWithForeignKeyIdentifier(QueryBuilder $query
      */
     public static function hasRootEntityWithCompositeIdentifier(QueryBuilder $queryBuilder, ManagerRegistry $managerRegistry) : bool
     {
+        return self::hasRootEntityWithIdentifier($queryBuilder, $managerRegistry, false);
+    }
+
+    private static function hasRootEntityWithIdentifier(QueryBuilder $queryBuilder, ManagerRegistry $managerRegistry, $foreign) : bool
```
Comment (theofidry) [inline src/Bridge/Doctrine/Orm/Util/QueryChecker.php]: `bool $foreign`?

Comment (theofidry) [inline src/Bridge/Doctrine/Orm/Util/QueryChecker.php]: `$isForeign`?

File: src/Bridge/Doctrine/Orm/Metadata/Property/DoctrineOrmPropertyMetadataFactory.php (hunk 1 above)
Comment (theofidry) [inline src/Bridge/Doctrine/Orm/Metadata/Property/DoctrineOrmPropertyMetadataFactory.php]:
```php
$writable = $doctrineClassMetadata instanceof ClassMetadataInfo
    ? $doctrineClassMetadata->isIdentifierNatural()
    : false
;
```

but I guess it violates our CS...

**Comment by Simperfit** [comment]
👍

**Pull request merged**